doc/installation/index.rst

```diff
@@ -152,7 +152,14 @@ compile NEST from source, see section** :ref:`advanced_install`.
 
              conda create --name ENVNAME -c conda-forge nest-simulator=*=mpi_openmpi*
 
-          The syntax for this install follows the pattern: ``nest-simulator=<version>=<build_string>``
+          The syntax for this install follows the pattern:
+          ``nest-simulator=<version>=<build_string>``. Build strings can be
+          found by browsing the `conda forge file list
```

Comment:
```suggestion
          found by listing the available versions with
          ```
          conda search -c conda-forge nest-simulator
          ```
          or by browsing the `conda forge file list
```

Reply:
Somehow I wasn't even aware of that command! Yes, that's definitely the way to do it.